### PR TITLE
UHM-4934: Create "circular workflow" for adding Lab Orders from the L…

### PR DESCRIPTION
**Pull request by mogoodrich**
https://pihemr.atlassian.net/browse/UHM-4934

### CODE REVIEW

File: app/js/components/orderEntry/OrderEntryPage.jsx [hunk 1]
```diff
@@ -58,10 +60,20 @@ export class OrderEntryPage extends PureComponent {
...
+    const { afterAddOrderUrl } = this.state;
     const orderCreatedMsg = intl.formatMessage({ id: "app.orders.create.success", defaultMessage: "Order Successfully Created" });
     if (added && labOrderData !== prevProps.createOrderReducer.labOrderData) {
       successToast(orderCreatedMsg);
-      this.props.fetchLabOrders(null, this.props.patient.uuid);
+      // if an afterAddOrderUrl has been specified, redirect to that page
+      if (afterAddOrderUrl && labOrderData && labOrderData.orders) {
+        // we pass in the uuid of the new order (just picking the "first" order if multiple were created)
+        let url = Handlebars.compile(afterAddOrderUrl)({
+          order: labOrderData.orders[0].uuid,
```
Comment (brandones) [inline app/js/components/orderEntry/OrderEntryPage.jsx:71]: Ok as long as you're confident that `labOrderData.orders` cannot be an empty array, and that if it has a member, that member must have a `uuid` element.

**Review by brandones — APPROVED** [review]

**Comment by brandones** [comment]
Oh yeah, gotta fix the build too. Looks like you got a couple lint failures. Use `npm run lint` locally and fix stuff.

File: app/js/components/orderEntry/OrderEntryPage.jsx (hunk 1 above)
Comment (mogoodrich) [reply app/js/components/orderEntry/OrderEntryPage.jsx:71]: I think so, but, fair enough, I'll add a check.

Comment (mogoodrich) [reply app/js/components/orderEntry/OrderEntryPage.jsx:71]: Actually, it really *must* have a uuid, but good to check that is has a member.

**Comment by mogoodrich** [comment]
Fixed the lint issues... merging, thanks @brandones !

**Pull request merged**